### PR TITLE
Temporarily Hide Phone Numbers

### PR DESCRIPTION
**Pull request by aarontitus**
This hotfix temporarily hides all user, contact, and organizational
phone numbers, due to call center staff sharing it with the public. See:
http://blog.crisiscleanup.org/2017/09/internal-phone-numbers-hidden.html

### CODE REVIEW

File: app/javascript/history.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Vue.filter('phone', function (phone) {
-  return phone.replace(/[^0-9]/g, '')
-    .replace(/(\d{3})(\d{3})(\d{4})/, '($1) $2-$3');
+  return phone.replace(.,''); //Temporarily hide phone number until we can re-train phone center staff
```
Comment (tsanders) [inline app/javascript/history.js]: Revert these changes

Comment (tsanders) [reply app/javascript/history.js]: Change the line here instead - https://github.com/CrisisCleanup/crisiscleanup/blob/master/app/javascript/components/history/HistoryVue.vue#L6

Comment (tsanders) [reply app/javascript/history.js]: Specfically remove this part of the line- `, {{claimedByUser.u_mobile | phone}}`

**Comment by arroyoDev** [comment]
Tested the site in staging.  Everything appears to be working properly.   Ready to push live.
